src/zzz_od/application/hollow_zero/lost_void/lost_void_app.py

```
@@ -237,14 +237,17 @@ def matrix_select_team(self) -> OperationRoundResult:
             predefined_idx = 0
         self.ctx.lost_void.predefined_team_idx = predefined_idx
         team_name = self.ctx.team_config.team_list[predefined_idx].name
+        # 移除配队名及ocr结果中的空白字符，提升文本匹配兼容性
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)